### PR TITLE
{lib}[intel/2018b] mpi4py v3.0.1

### PR DESCRIPTION
**Pull request by wpoely86**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by wpoely86** [comment]
Test report by @wpoely86
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
nic3 - Linux centos linux 7.5.1804, AMD Opteron(tm) Processor 6134, Python 2.7.5
See https://gist.github.com/6c1ba05106e4ac38c2a67045e886d51d for a full test report.

File: easybuild/easyconfigs/m/mpi4py/mpi4py-3.0.1-intel-2018b-Python-3.6.6.eb [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+ the Python programming language, allowing any Python program to exploit multiple processors."""
+
+toolchain = {'name': 'intel', 'version': '2018b'}
+
+source_urls = [BITBUCKET_DOWNLOADS]
+sources = [SOURCE_TAR_GZ]
+checksums = ['6549a5b81931303baf6600fa2e3bc04d8bd1d5c82f3c21379d0d64a9abcca851']
+
+dependencies = [('Python', '3.6.6')]
+
```
Comment (akesandgren) [inline easybuild/easyconfigs/m/mpi4py/mpi4py-3.0.1-intel-2018b-Python-3.6.6.eb]: As travis says, add "download_dep_fail = True"

**Comment by wpoely86** [comment]
Test report by @wpoely86
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
nic425 - Linux centos linux 7.5.1804, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz, Python 2.7.5
See https://gist.github.com/69a981c41f255141fbe500c120d55a1a for a full test report.

**Comment by wpoely86** [comment]
Test report by @wpoely86
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
nic400 - Linux centos linux 7.5.1804, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz, Python 2.7.5
See https://gist.github.com/822a19b273918bb98cd95ea042be0998 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node3106.skitty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 2.7.5
See https://gist.github.com/8c7cdee254f614e49054fbb36675d7d7 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2432.golett.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/f5360c0e7717bd68b3c6a9fbce6c156e for a full test report.

**Comment by boegel** [comment]
Going in, thanks @wpoely86!

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2062.delcatty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/2036fa437aa86c562cd05f1e0b390d5d for a full test report.

**Pull request merged**